frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseGitMacheteRepositoryReadyAction.java

```diff
@@ -3,12 +3,20 @@
 import com.intellij.openapi.actionSystem.AnAction;
 import com.intellij.openapi.actionSystem.AnActionEvent;
 import com.intellij.openapi.project.DumbAwareAction;
+import lombok.CustomLog;
 import org.checkerframework.checker.guieffect.qual.UIEffect;
 
 import com.virtuslab.gitmachete.frontend.actions.expectedkeys.IExpectsKeyGitMacheteRepository;
+import com.virtuslab.logger.IEnhancedLambdaLogger;
 
+@CustomLog
 public abstract class BaseGitMacheteRepositoryReadyAction extends DumbAwareAction implements IExpectsKeyGitMacheteRepository {
 
+  @Override
+  public IEnhancedLambdaLogger log() {
```

Comment:
Same here, should rather be defined in concrete classes (unless there's a good reason to keep it here)